tests/test_holoclean.py

```diff
@@ -32,9 +32,9 @@ def get_tid(row):
     return row['tid'] - 1
 
 def get_attr(row):
-    return row['attribute'].lower()
+    return row['attribute']
```

Comment:
This file conflicts now, but we shouldn't even need to fix it since the new eval API takes the attribute names without the need to lower them